glue/config.py

```diff
@@ -35,6 +34,7 @@ class Registry(object):
 
     def __init__(self):
         self._members = []
+        self._plugins = []
```

Comment:
Oh sorry -- I meant to type `_lazy_members`. I don't have any opinions on whether it should be public or private though.